android/src/main/java/io/radar/react/RNRadarModule.java

```diff
@@ -578,6 +579,7 @@ public void searchPlaces(ReadableMap optionsMap, final Promise promise) {
         }
         int radius = optionsMap.hasKey("radius") ? optionsMap.getInt("radius") : 1000;
         String[] chains = optionsMap.hasKey("chains") ? RNRadarUtils.stringArrayForArray(optionsMap.getArray("chains")) : null;
+        Map<String, String> chainMetadata = optionsMap.hasKey("chainMetadata") ? RNRadarUtils.stringStringMap(optionsMap.getMap("chainMetadata")) : null;
```

Comment:
If `stringStringMap()` returns `null` when the value passed in is `null`, do you even need the `hasKey()` check?

Reply:
ok i will update